core/ontology/generis.rdf

```diff
@@ -352,6 +352,11 @@
     <rdfs:label xml:lang="en-US"><![CDATA[User Role]]></rdfs:label>
     <rdfs:comment xml:lang="en-US"><![CDATA[The user centric Role Class]]></rdfs:comment>
   </rdf:Description>
+  <rdf:Description rdf:about="http://www.tao.lu/Ontologies/generis.rdf#ItemRole">
```

Comment:
Why the Item Role is created here and not in tao-core? https://github.com/oat-sa/tao-core/pull/2852/files#diff-f3f0e89bacd2a50449488338b4a72a02a696fa062af4269e99e90bd5b9aac6bcR214

We need to agree in one place (or better, have this configured, so the code is agnostic)